src/renderers/shared/shared/ReactTreeTraversal.js

```diff
@@ -90,13 +91,54 @@ function getParentInstance(inst) {
 /**
  * Simulates the traversal of a two-phase, capture/bubble event dispatch.
  */
+
+function isInteractive(inst) {
+  var tag = ReactDOMFeatureFlags.useFiber ? inst.type : inst._tag;
+
+  return (
+    tag === 'button' || tag === 'input' ||
+    tag === 'select' || tag === 'textarea' ||
+    tag === 'fieldset'
+  );
+}
+
+function shouldIgnoreElement(inst) {
+  if (inst && isInteractive(inst)) {
+    if (ReactDOMFeatureFlags.useFiber) {
```

Comment:
This is not sufficient because we can use ReactDOM and ReactDOMFiber at the same time when the flag is not on.